### PR TITLE
[TOL] getOutputType Attribute added

### PR DESCRIPTION
**Pull request by jbathmann**
This allows to access the outputtyp for subroutines.

### CODE REVIEW

**Comment by jbathmann** [comment]
This PR does not do the full trick. It makes the information on the outputdir accessible, if the output can be successfully initiated.
This is only possible, if the file exists and if it is empty... 
@jvollhueter maybe you can assume a general and empty outputdir like /pyMANGA/test/testoutputs (./test/testoutputs in the prj. files), which is always cleaned after a test has been performed (and output has been written). For this, you would need the output case accessor provided here...

File: TreeOutputLib/OneTimestepOneFile/OneTimestepOneFile.py [hunk 1]
```diff
@@ -116,3 +116,7 @@ def checkRequiredKey(self, key, args):
             raise KeyError("Key '" + key + "' in project file at position " +
                            "MangaProject__tree_output needs to be specified.")
         return tmp.text
+
+    ## This function returns the output directory
+    def getOutputDir(self):
+        return self.output_dir
```
Comment (jbathmann) [inline TreeOutputLib/OneTimestepOneFile/OneTimestepOneFile.py:122]:
```suggestion
        return self.output_dir
        
```

File: TreeOutputLib/NONE/NONE.py [hunk 2]
```diff
@@ -16,3 +16,7 @@ def __init__(self, args):
 
     def writeOutput(self, tree_groups, time):
         pass
+
+    ## This function returns the output directory
+    def getOutputDir(self):
+        return ""
```
Comment (jbathmann) [inline TreeOutputLib/NONE/NONE.py:22]:
```suggestion
        return ""

```

**Comment by jvollhueter** [comment]
Successfully tested.

**Pull request merged**